src/main/java/com/onarandombox/MultiverseCore/destination/DestinationsProvider.java

```diff
@@ -65,8 +56,8 @@ private void registerDestinationPerms(@NotNull Destination<?> destination) {
                                                            @Nullable String deststring
     ) {
         return destinationMap.values().stream()
-                .filter(destination -> issuer.hasPermission(PERMISSION_PREFIX + "self." + destination.getIdentifier())
-                        || issuer.hasPermission(PERMISSION_PREFIX + "other." + destination.getIdentifier()))
+                .filter(destination -> this.plugin.getPermissionsTool().hasDestinationTeleportPermission(null, issuer.getIssuer(), destination)
+                        || this.plugin.getPermissionsTool().hasDestinationTeleportPermission(issuer.getIssuer(), this.plugin.getServer().getConsoleSender(), destination))
```

Comment:
Would it make sense to have an overloaded method that takes 1 less param? Also, would it make sense to move the boolean logic into method? (Pretend I don't know what `hasDestinationTeleportPermission` does - which atm is true)

Reply:
lol i actually have an overload alr, smh didnt use it